scalafmt-tests/src/test/resources/test/IndentYieldKeyword.stat

```diff
@@ -47,11 +47,26 @@ yield
 >>>
 val sizes =
   for (row <- table)
+  yield (for (cell <- row)
+         yield
+           if (cell == null) 0
+           else cell.toString.length)
+<<< break before `for` 2
+val sizes = for (row <- table) yield (
+  for (cell <- row)
   yield
-    (for (cell <- row)
-     yield
-       if (cell == null) 0
-       else cell.toString.length)
+    if (cell == null) 0
+    else cell.toString.length
+)
+>>>
+val sizes =
+  for (row <- table)
+  yield (
```

Comment:
@lihaoyi notice that you need to manually insert a line break after `(` and before `)` to trigger this behavior, otherwise it defaults to the kinda funky output above
```
val sizes =
  for (row <- table)
  yield (for (cell <- row)
         yield
           if (cell == null) 0
           else cell.toString.length)
```